### PR TITLE
PLAT-73522: Change ContextualPopup appearance to match new designs

### PR DESCRIPTION
**Pull request by viodragon2**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed

* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Update contextual popup GUI

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- change border radius
- change border color
- change border width
- adjust arrow SVG accordingly to match the new border width
- add drop shadow

Enact-DCO-1.0-Signed-off-by: Stephen Choi <stephen.choi@lge.com>

### CODE REVIEW

File: packages/moonstone/styles/variables.less [hunk 1]
```diff
@@ -351,8 +351,8 @@
 
 // ContextualPopup
 // ---------------------------------------
-@moon-contextual-popup-border-width: @moon-button-border-width;
-@moon-contextual-popup-border-radius: 15px;
+@moon-contextual-popup-border-width: 1px;
```
Comment (ryanjduffy) [inline packages/moonstone/styles/variables.less]: `1px` is negligible at higher resolutions (4k and 8k) and isn't currently adjusted by our RI plugin. I think we'll need to resolve this eventually but not necessarily in this PR as it's a broader issue than is being solved here.

File: packages/moonstone/ContextualPopupDecorator/ContextualPopupDecorator.js [hunk 2]
```diff
@@ -615,9 +617,7 @@ const Decorator = hoc(defaultConfig, (config, Wrapped) => {
 							<PopupComponent {...popupPropsRef} />
 						</ContextualPopupContainer>
 					</FloatingLayer>
-					<div ref={this.getClientNode}>
-						<Wrapped {...rest} />
-					</div>
+					<Wrapped ref={this.getClientNode} {...rest} />
 				</div>
```
Comment (ryanjduffy) [inline packages/moonstone/ContextualPopupDecorator/ContextualPopupDecorator.js]: Also unrelated to this PR but I wonder if we really need this wrapping DOM. The only CSS that is applied here is `display: inline-block` and I'm not sure why that is even required. Appears this was ported from Enyo (https://github.com/enyojs/moonstone/commit/9b5ac029913e5804f146dd9c80b7cd2af0e5a938) but the rationale might not be relevant any longer.

File: packages/moonstone/styles/colors-light.less [hunk 3]
```diff
@@ -78,7 +78,8 @@
 // ContextualPopup
 // ---------------------------------------
 @moon-contextual-popup-bg-color: @moon-bg-color;
-@moon-contextual-popup-border-color: fade(@moon-black, 50%);
+@moon-contextual-popup-border-color: #bbb;
```
Comment (ryanjduffy) [inline packages/moonstone/styles/colors-light.less]: Any insight on why we need a new color here (`#bbbbbb`) vs using `@moon-gray` (`#a6a6a6`)?

Comment (viodragon2) [reply packages/moonstone/styles/colors-light.less]: #bbbbb is a new color for border colors. #a6a6a6 is close, but we decided to keep it separate.

File: packages/moonstone/ContextualPopupDecorator/ContextualPopupDecorator.js (hunk 2 above)
Comment (viodragon2) [reply packages/moonstone/ContextualPopupDecorator/ContextualPopupDecorator.js]: The CSS part might be irrelevant now, but we still need `ref` in order to calculate the position of the popup.

File: packages/moonstone/styles/variables.less (hunk 1 above)
Comment (viodragon2) [reply packages/moonstone/styles/variables.less]: This is intentional. The border isn't considered as a GUI element, but rather a functional "line" that differentiates between the popup and the outside.

File: packages/moonstone/ContextualPopupDecorator/ContextualPopupDecorator.js (hunk 2 above)
Comment (ryanjduffy) [reply packages/moonstone/ContextualPopupDecorator/ContextualPopupDecorator.js]: I don't think I was clear in my comment. The "wrapping div" to which I was referring wasn't the removed one but the one around Wrapped and FloatingLayer. No concerns with the proposed change.

**Pull request merged**